### PR TITLE
github info and reformatting

### PR DESCRIPTION
**Pull request by rosemccollum**

### CODE REVIEW

File: docs/git-commands.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    * `git branch new_branch_name` also creates a new branch but doesn't also move you to the new branch
+
+`git add file_or_folder`
+
+- Add changes from a specific file/folder to your commit. To add multiple files at once there are several options:
+
+    * `-A` to add all new, modified, and deleted files
+
+    * `.` to add all new and modified files but not deleted files
```
Comment (tikal004) [inline docs/git-commands.md:27]: the '.' does stage file deletion

File: docs/git-commands.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Common Github Commands
```
Comment (tikal004) [inline docs/git-commands.md:1]: git is an open source software that github leverages for its platform. These are git commands, not github commands.

File: docs/git-commands.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+This page contains common commands used to work with Github repositories. This is by no means an exhaustive list of commands. 
+
+## Basics 
+
+`git clone url_to_repo`
```
Comment (tikal004) [inline docs/git-commands.md:7]: its worth discussing the difference between https and ssh cloning here

File: docs/git-commands.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+If a branch name changed and you can no longer push to/pull from the old branch name:
+
+```
+git branch -m old_name new_name
+git fetch origin
+git branch -u origin/new_name new_name
+git remote set-head origin -a
+```
+
```
Comment (tikal004) [inline docs/git-commands.md:62]: Adding a new branch to the origin that you made locally
git push --set-upstream origin branch-name

File: docs/git-commands.md (hunk 1 above)
Comment (rosemccollum) [reply docs/git-commands.md:27]: smh stack overflow lied

File: docs/git-commands.md (hunk 4 above)
Comment (rosemccollum) [reply docs/git-commands.md:62]: this needs to be done after this set of commands? this set of commands is when the name on github changed but hasn't been updated locally yet

Comment (tikal004) [reply docs/git-commands.md:62]: Not directly related, just another useful one that I use a lot.

**Review by tikal004 — APPROVED** [review]

**Pull request merged**